setup.py

```diff
@@ -31,7 +31,9 @@
       author_email="jelmer@jelmer.uk",
       license="GNU GPLv3 or later",
       url="https://www.jelmer.uk/projects/xandikos",
-      install_requires=['icalendar', 'dulwich', 'defusedxml'],
+      install_requires=[
+          open('requirements.txt').readlines()
```

Comment:
Can we drop requirements.txt instead? Reading https://packaging.python.org/requirements/#requirements-files it serves a different role, and I don't think it's actually necessary in this case.